dev/CommonStyles/DateTimePickerFlyout_themeresources.xaml

```diff
@@ -189,14 +197,11 @@
                         <Border
                             x:Name="RootBorder"
                             contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
-                            BorderBrush="{ThemeResource DateTimePickerFlyoutButtonBorderBrush}"
-                            BorderThickness="{ThemeResource DateTimeFlyoutButtonBorderThickness}"
                             Padding="{TemplateBinding Padding}"
-                            AutomationProperties.AccessibilityView="Raw"
-                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                            contract7NotPresent:CornerRadius="{TemplateBinding ControlCornerRadius}">
+                            AutomationProperties.AccessibilityView="Raw" >
                             <TextBlock
                                 x:Name="Content"
+                                Foreground="{TemplateBinding Foreground}"
```

Comment:
>Foreground [](http://example.com/codeflow?start=61&length=10)

Is this needed? it should propogate down automatically.

Reply:
It did not - worked with Mark on this one, definitely needed in order for the default arrow state to pick up the desired color.